.gitignore

```diff
@@ -14,15 +14,6 @@ node_modules/
 .cache
 dist/
 
-# packages
-/hex
-/hsl
-/hslString
-/hsv
-/rgb
-/rgbString
-/HexInput
```

Comment:
**Discussion:**

I feel that this update is going to make the top-level repo structure more complex.
And that'll make the barrier to entry way higher.
I just thought that keeping so many folders in repo root isn't the best idea.

It seems to me that we could combine `build-packages.js` solution and the one you did. I mean to create the same package.json, dist subfolders, run the same command etc. I'm going to create a separate PR to try.

Reply:
Fair enough. Though I still think this will just move the complexity into your tooling instead. It'll also be slower, though depending on what you're used to, might not be important. 

You'll just need to first generate a valid `package.json`, then run the Microbundle in that directory, rather than the other way around.